Esri/arcgis-maps-sdk-swift-samples
# [New] Add raster from service

## Description

This PR implements **`Add raster from service `** in `Layers` category.

`swift/issues/5458`

## How To Test

Load view, and wait for raster download to complete. Map should zoom in San Fransisco Golden Gate area of map.

## Screenshots
![add-raster-from-service](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/77968d1c-dd60-4308-8f83-c4f82fc0f396)
